README.md

```diff
@@ -41,7 +41,7 @@ with systemd.
 
 ### Requirements
 
-* Puppet 4.6.1 or later.  Puppet 3 was supported up until release 0.2.0.
+* Puppet 4.6.1 or later.
```

Comment:
Looks like it's really 5.5.8 or later.

https://github.com/voxpupuli/puppet-chrony/blob/679302f6ff7f13a196e26c23492a08259158922c/metadata.json#L49

Given it's in metadata.json, is this really needed at all?

Reply:
Makes sense to me, I'll adjust.